radio/views/feedback.py

```diff
@@ -0,0 +1,16 @@
+from aiohttp import web
+import radio.services.mail
+
+
+class Collection(web.View):
+    """
+    API view that is used for email feedback post handling
+    """
+
+    async def post(self):
+        data = await self.request.post()
+        mail_data = {'sender': data['sender_email'], 'body': data['body']}
```

Comment:
Front end will not allow empty values. You can check it in EmailModalcomponent. But yeah, it make sense also to validate this on backend. it would be nice to have something like Django forms for this purpose. Will update.